source/module_hamilt_pw/hamilt_pwdft/fs_kin_tools.h

```diff
@@ -0,0 +1,72 @@
+#ifndef W_ABACUS_DEVELOP_ABACUS_DEVELOP_SOURCE_MODULE_HAMILT_PW_HAMILT_PWDFT_FS_KIN_TOOLS_H
+#define W_ABACUS_DEVELOP_ABACUS_DEVELOP_SOURCE_MODULE_HAMILT_PW_HAMILT_PWDFT_FS_KIN_TOOLS_H
```

Comment:
the name seems too long

Reply:
The previous CI would automatically format into that style, but it has been deprecated now. Indeed, it can be shorter.